lib/itc/post_data.rb

```diff
@@ -107,6 +107,32 @@ def update_app_data(config, localization)
           isEmptyValue: true,
           isRequired: false
         },
+        watchAppIcon: {
+          errorKeys: nil,
+          isEditable: true,
+          isRequired: false,
+          value: {
+            assetToken: nil,
+            originalFileName: nil,
+            sortOrder: nil,
+            thumbNailUrl: nil,
+            thumbnailAssetToken: nil,
+            url: nil
+          }
+        },
+        bundleInfo: {
```

Comment:
Yeah, I intentionally omitted them, but I don't think it hurts to add them back. Apple has changed non-required fields to required before